tasks/emissions.ts

```diff
@@ -179,9 +180,50 @@ task("revenue-donate-rewards").setAction(async (_, __, runSuper) => {
     await runSuper()
 })
 
+subtask("votium-forward", "Forwards votium bribe. from votium dial")
+    .addOptionalParam("speed", "Defender Relayer speed param: 'safeLow' | 'average' | 'fast' | 'fastest'", "average", types.string)
+    .addOptionalParam("proposal", "Convex finance proposal for Weekly Gauge Weight", undefined, types.string)
```

Comment:
this doesn't look like an optional param. You can change it to `addParam`

Reply:
Thanks @naddison36 , the task is ready for review